### ISSUE TITLE
remarks branch has error

### ISSUE THREAD

**Issue by einalex** [samuelyeewl/zotrm#9]
```
Traceback (most recent call last):
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 537, in <module>
    main(args.verbose, args.landscape, args.dry_run)
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 518, in main
    sent_papers = send_papers(zot, rmapi, config, verbose=verbose, landscape=landscape,
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 446, in send_papers
    attachments = get_pdf_attachments(paper, zot, config)
  File "/home/user/git/projects/Remarkable/zotrm/zotrm/zotrm.py", line 139, in get_pdf_attachments
    foundfiles = list(filter(r.match, pdflist))
NameError: name 'pdflist' is not defined
```

**Comment by einalex** [comment]
maybe you want something like `foundfiles = glob.glob(os.path.join(dirname,"**/", filename), recursive = True)` ?

**Comment by samuelyeewl** [comment]
Thanks for the catch, this was a bug introduced when I refactored the code to use linked base attachment directories. Fixed in e2fb6e1.
